test/e2e/metering_manual_install_test.go

```diff
@@ -107,6 +110,105 @@ func testManualMeteringInstall(
 	assert.NoError(t, err, "capturing logs and uninstalling metering should produce no error")
 }
 
+func createNFSProvisioner(ctx *deployframework.DeployerCtx) error {
+	//Waiting for Metering pods
+	ctx.TargetPodsCount = nonHDFSTargetPodCount
+
+	_, err := createTestingNamespace(ctx.Client, "nfsTestingNamespace")
+	if err != nil {
+		return err
+	}
+
+	files := map[string]string{
+		"pod":          "/test/testhelpers/nfs_server.yaml",
+		"service":      "/test/testhelpers/nfs_service.yaml",
+		"storageClass": "/test/testhelpers/nfs_storageclass.yaml",
```

Comment:
This needs to be referencing the root of the directory. I'm not sure if the `deployframework.DeployerCtx` type has knowledge of the root directory, but the `deployframework.DeployFramework` would. Check if there's any existing fields in the ctx structure that points to the path of the testing resources.